src/software/ai/hl/stp/play/defense/defense_play_test.py

```diff
@@ -81,8 +162,10 @@ def setup(*args):
                 )
             ]
         ],
-        ag_eventually_validation_sequence_set=[[]],
-        test_timeout_s=30,
+        ag_eventually_validation_sequence_set=[
```

Comment:
you might be able to put these validations in the `inv_always_validation_sequence_set` and `inv_eventually_validation_sequence_set` instead of the `ag_` ones